core/src/test/scala/org/apache/spark/deploy/StandaloneDynamicAllocationSuite.scala

```diff
@@ -466,6 +466,23 @@ class StandaloneDynamicAllocationSuite
     }
   }
 
+  test("kill all executors on localhost") {
```

Comment:
this doesn't really have anything to do with Dynamic Allocation, so this is a strange suite to put this test in.  Though it does seem to have a useful framework -- perhaps there is a base class to pull out?